src/action-ros-ci.ts

```diff
@@ -803,12 +813,14 @@ done`;
 			],
 			options,
 		);
-		await execShellCommand(
-			[
-				`/usr/bin/git config --global --unset-all url.https://x-access-token:${importToken}@${gihubServerDomain}/.insteadof`,
-			],
-			options,
-		);
+		if (!gitSshPort) {
```

Comment:
I had to add this condition following after https://github.com/ros-tooling/action-ros-ci/pull/998 being merged.